react/react-fc-2-users-app/users-app/src/components/accordion/accordion.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+
+import './accordion.css';
+
+export const Accordion = ({
```

Comment:
The second exercise was with lifting the state up. Did you have any issues here?

Reply:
@atodoroski I moved the state in user.js and wrote this in accordion.js :  
 `const handleSetIndex=(index)=>(!isOpened!== index) && setIsOpened(index)` , it didn't work properly but then I saw the solution in the Ajax in React class. I will fix it.

Reply:
ok please try it